packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -895,6 +896,12 @@ class DropdownButton<T> extends StatefulWidget {
   /// The callback will not be invoked if the dropdown button is disabled.
   final VoidCallback onTap;
 
+  ///Called when the dropdown is closed, regardless of if a selection was made.
```

Comment:
This and all the docs
```suggestion
  /// Called when the dropdown is closed, regardless of if a selection was made.
```